custom_components/frigate/services.yaml

```diff
@@ -1,4 +1,40 @@
 ---
+export_recording:
+  name: Export recording
+  description: Export a custom recording or timelapse.
+  target:
+    entity:
+      integration: frigate
+      domain: camera
+      device_class: camera
+  fields:
+    playback_factor:
+      name: Playback Factor
+      description: Playback factor for recordings
+      required: true
+      advanced: false
+      example: Realtime
+      default: Realtime
+      selector:
+        select:
+          options:
+            - "Realtime"
```

Comment:
I'd argue it's a little more typical / less suprising to use lowercase only for the options (and remove the call to `lower()` in the API code).

Reply:
Works for me 👍